source/_integrations/pi_hole.markdown

```diff
@@ -63,3 +63,7 @@ data:
 target:
   entity_id: all
 ```
+
+## Switches
+
+The integratation will create a switch for the pi-hole named `switch.pi_hole_FRIENDLYNAME` which will allow you to toggle the pi-hole on and off.
```

Comment:
Entity IDs are not guaranteed and can be renamed. Just describe the switch and leave the entity ID out.

Reply:
will do

Reply:
Done